plugins/inputs/haproxy/sample.conf

```diff
@@ -10,10 +10,10 @@
   # password = "admin"
 
   ## If no servers are specified, then default to 127.0.0.1:1936/haproxy?stats
-  servers = ["http://myhaproxy.com:1936/haproxy?stats"]
+  servers = ["http://myhaproxy.com:1936/haproxy?stats", "tcp://127.0.0.1:1936"]
 
   ## You can also use local socket with standard wildcard globbing.
-  ## Server address not starting with 'http' will be treated as a possible
+  ## Server address not starting with 'http://', 'https://', 'tcp://' will be treated as a possible
   ## socket, so both examples below are valid.
   # servers = ["socket:/run/haproxy/admin.sock", "/run/haproxy/*.sock"]
```

Comment:
The sample config values should indicate the default value. Most users tend to stop reading examples once they have come across one, so the second paragraph showing the socket examples, is often ignored or missed. Let's combine these two sections into something like:

```
  ## URLs to gather stats from
  ## For HTTP and HTTPS URLs, the URL URL must specified the complete path to
  ## the endpoint containing stats:
  ##   - http://myhaproxy.com:1936/haproxy?stats
  ##   - https://myhaproxy.com:8000/stats
  ## The plugin can also connect to a socket:
  ##   - socket:/run/haproxy/admin.sock
  ##   - tcp://127.0.0.1:1936
  ##   - /run/haproxy/*.sock
  # servers = ["http://127.0.0.1:1936/haproxy?stats"]
```

Reply:
Thanks! Good points!